cli_tools/gce_windows_upgrade/main.go

```diff
@@ -0,0 +1,100 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// gce_windows_upgrade is a tool for upgrading GCE Windows instances.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/logging/service"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_windows_upgrade/upgrader"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+)
+
+var (
+	clientID               = flag.String(upgrader.ClientIDFlagKey, "", "Identifies the upgrade client. Set to `gcloud`, `api` or `pantheon`.")
+	project                = flag.String("project", "", "Project containing the instance to upgrade. Can be overridden if the -instance parameter specifies a different project.")
+	zone                   = flag.String("zone", "", "Zone containing the instance to upgrade. Can be overridden if the -instance parameter specifies a different zone.")
+	instance               = flag.String("instance", "", "Instance to upgrade. Can be either the instance name or the project path to the instance, for example: 'projects/<project>/zones/<zone>/instances/<instance>'.")
+	skipMachineImageBackup = flag.Bool("skip-machine-image-backup", false, "Set to false to generate a backup for the instance. Set to true to prevent the instance from being backed up. True is not recommended unless you already have a backup for the instance.")
```

Comment:
Zoran I see what you are saying but at Google we use "you" to avoid using passive voice, which omits the actor. I agree though that we should not overuse "you".

How about: "Set to false to generate a backup for the instance. Set to true to prevent generation of a backup instance. True is not recommended unless there is already a backup of the instance."

Reply:
Changed as suggested